app/views/RoomActionsView/index.js

```diff
@@ -452,14 +487,48 @@ class RoomActionsView extends React.Component {
 			});
 		}
 
+		// If can edit this room
+		// If this room type can be Encrypted
+		// If e2e is enabled for this server
+		if (canEdit && E2E_ROOM_TYPES[t] && e2eEnabled) {
+			// Insert after jitsi section
```

Comment:
We need to refactor this view soon, but this kind of comment only makes it worse :)
What if we don't refactor and Jitsi is moved around?
This comment may never be updated 😛